src/Microsoft.Diagnostics.Runtime/ClrRuntime.cs

```diff
@@ -831,12 +831,12 @@ public RuntimeBase(ClrInfo info, DataTargetImpl dataTarget, DacLibrary lib)
             _dacInterface.Flush();
 
             IGCInfo data = GetGCInfo();
```

Comment:
I don't think this is the right fix.  This check (GetGCInfo returning null) for desktop CLR is checking for a bad crash dump.  Basically we always expect that we can get the GCInfo out of a crash dump, otherwise that means the crash dump was generated incorrectly.  So this is a real and valid exception in that case.  From you changing it, I assume that means that this returns null in .Net Native sometimes?

Instead of this change, I would first suggest debugging the dac when GetGCInfo returns null, and see why that happens and if it's expected.  If you are 100% sure it's ok that .Net Native dac can return null here and that's not a bug, then removing "if (data == null)" should only be done by 'NativeRuntime'.

(And if you do that, I would suggest pulling the code dealing with IGCInfo here into a virtual function called 'InitGCInfo' that NativeRuntime overrides.)